pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -3180,3 +3203,11 @@ func Test_isPodWorthRequeuing(t *testing.T) {
 		})
 	}
 }
+
+func mustAddUnschedulableIfNotPresent(t *testing.T, q *PriorityQueue, logger klog.Logger, pInfo *framework.QueuedPodInfo, podSchedulingCycle int64) {
```

Comment:
To avoid ↓ in all places.

```
ERROR: pkg/scheduler/internal/queue/scheduling_queue_test.go:1559:32: Error return value of `q.AddUnschedulableIfNotPresent` is not checked (errcheck)
ERROR: 	q.AddUnschedulableIfNotPresent(logger, q.newQueuedPodInfo(highPriorityPodInfo.Pod, "plugin"), q.SchedulingCycle())
```

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/119105/pull-kubernetes-linter-hints/1700123530751905792